overlord/aspectstate/aspectstate.go

```diff
@@ -93,6 +93,15 @@ func GetAspect(st *state.State, account, bundleName, aspect string, fields []str
 		return nil, err
 	}
 
+	if len(fields) == 0 {
+		val, err := asp.Get(tx, "")
+		if err != nil {
+			return nil, err
+		}
+
+		return map[string]interface{}{"all": val}, nil
```

Comment:
Why introduce the `"all"`, just to strip it away when this function is called in api_aspects.go?

Reply:
It was just to meet the return type since `val` may not be a map. I forgot to go back to it, thanks for pointing it out